subliminal/providers/itasa.py

```diff
@@ -0,0 +1,390 @@
+# -*- coding: utf-8 -*-
+import copy
+
+import io
+
+import logging
+
+from babelfish import Language
+
+from requests import Session
+
+from zipfile import ZipFile, is_zipfile
+
+from . import ParserBeautifulSoup, Provider, get_version
+
+from .. import __version__
+
+from .. cache import SHOW_EXPIRATION_TIME, region
+
+from .. exceptions import AuthenticationError, ConfigurationError, TooManyRequests
+
+from .. subtitle import (Subtitle, fix_line_ending, guess_matches, guess_properties, sanitize_string,
+                         sanitized_string_equal)
+
+from .. video import Episode
+
+logger = logging.getLogger(__name__)
```

Comment:
Which line do you mean? The validation with PEP8 is ok. if you can tell me the line I will copy style from addic7ed.